con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -163,11 +177,29 @@ public void setCountry(Country country) {
         this.country = country;
     }
 
-    @Override
-    public String toString() {
-        return "Id: " + this.getId() + ", Name: " + this.getName() + " , Rating: " + this.getRating() + " ,Address: " + this.getAddress() +
-                " ,Zip: " + this.getZip() + " ,City: " + this.getCity() +
-                " ,Phone: " + this.getPhone() + " ,Fax: " + this.getFax() + " ,Email: " + this.getEmail() + " ,Website: " + this.getWebsite() +
-                " ,Description: " + this.getDescription();
+    public Time getCheckIn() {
+        return checkIn;
+    }
+
+    public void setCheckIn(String checkIn) {
+        this.checkIn = timeCheck(checkIn);
+    }
+
+    public Time getCheckOut() {
+        return checkOut;
+    }
+
+    public void setCheckOut(String checkOut) {
+        this.checkOut = timeCheck(checkOut);
+    }
+
+    private Time timeCheck(String string) {
+        final String SECONDS = ":00";
+        final String REGEX = ("^([0-1]\\d|2[0-3]):([0-5]\\d)$");
+        if (string != null) {
```

Comment:
Instead of doing `null` check, throwing `IllegalArgumentException` would probably fit better. Logic of the method is unclear if you pass `null` and get `null`.